posthog/queries/session_recordings/session_recording_list.py

```diff
@@ -355,7 +355,7 @@ def _data_to_return(self, results: List[Any]) -> List[Dict[str, Any]]:
                 "matching_events": [
                     {
                         "events": [
-                            dict(zip(["timestamp", "id", "session_id", "window_id"], event)) for event in row[i + 1]
```

Comment:
renaming to uuid to remain consistent with matching events data returned from persons query